packages/app/src/app/pages/Dashboard/Header/index.tsx

```diff
@@ -121,6 +121,22 @@ export const Header: React.FC<HeaderProps> = React.memo(
             Create
           </Button>
 
+          <Button
+            variant="ghost"
+            css={css({ width: 'auto' })}
+            onClick={() => {
+              window.open('http://codesandbox.io/discover', '_blank');
```

Comment:
```suggestion
              window.open(`http://codesandbox.io${dashboardUrls.discover(activeTeam)}`, '_blank');
```

Reply:
@olarclara this will redirect to the website?

Reply:
dropped the commit that accepted the suggestion, my bad.